### ISSUE TITLE
Support non pass/fail messages

### ISSUE THREAD

**Issue by dennisjbell** [jhunt/shout#10]
Sometimes you want to always point out that something has reached a point in a pipeline (ie notification that something is ship-able or production is ready to be manually deployed)

While it is arguable that this is a "green" condition, you don't want the notification blocked by a previous "green" condition.

**Comment by dennisjbell** [comment]
Duplicate of #4
